src/details/ArborX_DetailsTags.hpp

```diff
@@ -45,6 +45,12 @@ struct Tag<Box>
   using type = BoxTag;
 };
 
+template <typename T>
```

Comment:
So does this let you just add a `Tag` typedef to any object and then you can extract the tag without having to write a specialization for that object?

Reply:
Correct.  I plan on changing the case `Tag` -> `tag` sometime soon for consistency.